index.html

```diff
@@ -17,6 +17,7 @@ <h1>Solide State</h1>
             <a href="">Log in</a>
             <a href="">Sign Up</a>
         </nav>
+        </header>
         <main>
             <section class="Hero">
```

Comment:
Please don't use the `uppercase` letter in classnames.
```suggestion
            <section class="hero">
```